doc/userguide/devguide/internals/engines/stream/applayertriggerrawreassembly.rst

```diff
@@ -0,0 +1,48 @@
+Triggering Raw Stream Reassembly from Application Layer
+-------------------------------------------------------
+
+Function ``AppLayerParserTriggerRawStreamReassembly``
```

Comment:
This looks a bit odd to me. Do you have any suggestions to incorporate the fn name better @jufajardini ?

Reply:
How about:

It is possible to trigger the reassembly of the raw stream from the application layer protocol parsers, by calling the function ``AppLayerParserTriggerRawStreamReassembly``.

(then proceed with the rest)?